proton-j/src/test/java/org/apache/qpid/proton/engine/impl/ssl/SimpleSslTransportWrapperTest.java

```diff
@@ -117,16 +117,6 @@ public void testInputIncompletePacketInThreeParts()
     }
 
     @Test
-    public void testUnderlyingInputUsingSmallBuffer_receivesAllDecodedInput() throws Exception
-    {
-        _underlyingInput.setInputBufferSize(1);
```

Comment:
I deleted this Test from this File and Moved it to a brand new file for 2 reasons:
1) This function has a bug - SetInputBufferSize - sets only the integer value. It doesn't change anything with actual Buffer Size.
2) Since, changing buffer size will change the behavior of SimpleSslTransportWrapper - this could impact other testcases in the same file. So, moved the Size related test to a separate file...